### PR TITLE
cuba progress indicator

### PR DESCRIPTION
**Pull request by JakovRus**

### CODE REVIEW

File: cuba-progress-indicator.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+          document.body.addEventListener('cuba-data-loading-start', this._incrementCounter.bind(this));
+          document.body.addEventListener('cuba-data-loading-finish', this._decrementCounter.bind(this));
+          document.body.addEventListener('cuba-data-loading-error', this._decrementCounter.bind(this));
+        }
+
+        disconnectedCallback() {
+          super.disconnectedCallback();
+
+          document.body.removeEventListener('cuba-data-loading-start', this._incrementCounter.bind(this));
```
Comment (web-devel) [inline cuba-progress-indicator.html]: It's not the same listener

**Comment by jreznot** [comment]
Please sign CLA before we review and merge this

File: cuba-progress-indicator.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+      #progress {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 0;
+        z-index: 500;
+        height: var(--progress-height, 3px);
+
+        background-color: var(--progress-color, #0040a8);
```
Comment (web-devel) [inline cuba-progress-indicator.html:18]: Describe vars in comments

File: cuba-progress-indicator.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    </style>
+
+    <div id="progress"></div>
+
+  </template>
+  <script>
+    {
+      /**
+       *  `<cuba-progress-indicator>` is a Polymer 2 element for data loading progress indication.
```
Comment (web-devel) [inline cuba-progress-indicator.html:67]: So we have to remove Polymer 1 from version range in `bower.json`.

**Review by web-devel — CHANGES_REQUESTED** [review]

**Pull request merged**